### DISCUSSION TITLE
Keycloak - Any way to enable Access Logs ?

### DISCUSSION THREAD

**Discussion by souravs17031999** [keycloak/keycloak#17617] [Q&A]
Hi Community, 

Is there a way to enable access logs in Keycloak ? AFAIK, We can only enable server application logs using 
```KC_LOG_LEVEL```

One way to do it is using Quarkus access logs using 
```QUARKUS_HTTP_ACCESS_LOG_ENABLED``` and 
```QUARKUS_HTTP_ACCESS_LOG_PATTERN```

But it would be nice if keycloak provides such mechanism to enable those logs which can help in getting some metrics based on client calls who calls ? from where ? how many times ? and many other insights such as status codes ? response length ? headers (in debug mode) ?

**Comment by steven-aerts** [comment]
It is explained here: https://github.com/keycloak/keycloak/issues/10618#issuecomment-1144827773
